core/src/main/java/org/pentaho/di/connections/vfs/provider/ConnectionFileObject.java

```diff
@@ -186,6 +186,11 @@ public InputStream getInputStream() throws FileSystemException {
     return requireResolvedFileObject().getInputStream();
   }
 
+  @Override
+  public InputStream getInputStream( int bufferSize ) throws FileSystemException {
+    return requireResolvedFileObject().getInputStream( bufferSize );
+  }
+
```

Comment:
Was missing this override, that certainly is not being called. Would fail before this override is added, for not delegating to the resolved file object.